### ISSUE TITLE
Storefront hardcodes the editor font-size to 16px

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/storefront#1337]
Storefront hardcodes the editor font-size to 16px, so users that change their browser default font-size will not see it applied in the editor. However, it's applied in the frontend, so that causes an incongruity between both.

## Screenshots
_Editor:_
![imatge](https://user-images.githubusercontent.com/3616980/80344348-b6511e80-8867-11ea-8dbd-4ae9cf63f68b.png)

_Frontend:_
![imatge](https://user-images.githubusercontent.com/3616980/80344368-bf41f000-8867-11ea-8897-63592270094e.png)

**Comment by amitraj2203** [comment]
Hi @Aljullu,

I was able to reproduce this issue. After checking the codebase, I believe removing the [following](https://github.com/woocommerce/storefront/blob/400457befbceaab218c586fbb16c53e4e362abc3/assets/css/base/gutenberg-editor.scss#L23-L24) lines would resolve it, the `line-height` is also part of the problem.

https://github.com/woocommerce/storefront/blob/400457befbceaab218c586fbb16c53e4e362abc3/assets/css/base/gutenberg-editor.scss#L23-L24

Let me know if my findings are correct or if I’m missing anything. I’d be happy to raise a PR and contribute to the fix.
Thanks!

**Comment by amitraj2203** [comment]
@Aljullu Sorry for the ping.
I've raised a PR to fix this [issue](https://github.com/woocommerce/storefront/pull/2189). Please review it when you get a chance.

**Comment by amitraj2203** [comment]
Hi @Aljullu,
Just a gentle reminder to take a look at [this PR](https://github.com/woocommerce/storefront/pull/2189) when you get a chance.

Thanks in advance! 😊
